package.json

```diff
@@ -123,5 +123,8 @@
   "workspaces": [
     "client/*",
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "@types/uuid": "^8.3.0"
+  }
```

Comment:
should we leave uuid into the package cuz we arent doing anything with uuid in client folder. _as far as i know of_

Reply:
addressed